index.html

```diff
@@ -2097,6 +2095,19 @@ <h4>Semantic Interoperability</h4>
 <code>@protected</code> feature in the JSON-LD 1.1 specification.
             </li>
           </ul>
+          <p>
+A human-readable document describing the expected order of values for the
+<code>@context</code> <a>property</a> SHOULD be available. A machine-readable
+description (i.e. a normal JSON-LD "context" document) SHOULD be available
+at the URL specified in the <code>@context</code> <a>property</a>.
+          </p>
+          <p>
+These requirements guarantee semantic interoperability between JSON and JSON-LD
+for terms defined by the <code>@context</code> mechanism. While JSON-LD processors
+will use the specific mechanism provided and can verify that all terms are correctly
+specified, JSON-based processors implicitly accept the same set of terms
+without testing that they are correct.
+          </p>
```

Comment:
If you expect JSON processors to check that the set of @context values are in the correct order, then why don't you expect JSON processors to check that all the terms in the VC are also defined by the @context? Only a human administrator can configure the JSON parser to check both of these. So why one without the other? Or have I misunderstood something?

Reply:
> then why don't you expect JSON processors to check that all the terms in the VC are also defined by the @context?

Frankly, because JSON developers have said that they don't want to do that. Experience w/ schema.org demonstrates that most will happily just copy-paste examples and work from those.

In an ideal world, everyone would just point a JSON-LD processor at the document and hit a button... but there is non-trivial subset of developers that don't want the "burden" of running a JSON-LD processor on their content and would rather risk semantic non-interop than pull an extra library into their software stack. In any case, we're trying to make that subset of JSON implementers happy... the ones that don't want to be bothered by semantics (but we still have to make sure that we provide enough tooling for them to interop when it becomes important to them.